src/collector.ts

```diff
@@ -8,6 +8,8 @@ import * as Xml2Object from 'xml2object';
 import { stream_from_string } from './utils';
 import { Stream } from 'stream';
 
+const semverRegex = require('semver-regex');
```

Comment:
Why can't this be imported?

Reply:
I tried a lot and also asked Darshan about this, but we could not find a way to import it. Need your help to do this.

Reply:
`import semverRegex = require('semver-regex');`